### PR TITLE
Release 3.4.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.4.1 (2019-08-07)
+
+- Fixed the repeater item component to properly align selction checkboxes and context menus. [#55](https://github.com/blackbaud/skyux-lists/pull/55)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: selection*

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Fixed the repeater item component to not show the collapse chevron when then the item cannot collapse. [#56](https://github.com/blackbaud/skyux-lists/pull/56)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the repeater item component to hide the collapse chevron when items cannot collapse.

Is that still accurate?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Ya. This was for a bug where there were times where even when "expandMode" was "none" the chevron would show up.

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md]: I worded my question poorly. I meant is my proposed edit accurate, and if yes, let's go with that please 😬

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

**Pull request merged**